.github/workflows/ci.yml

```
@@ -0,0 +1,83 @@
+name: Pulse CI
+on:
+  push:
+    branches-ignore:
+    - _**
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    container: mtzguido/dev-base
+    steps:
+      - run: echo "HOME=/home/user" >> $GITHUB_ENV
+      - uses: mtzguido/set-opam-env@master
+
+      - uses: actions/checkout@master
+        id: checkout-fstar
+        with:
+          path: FStar
+          repository: FStarLang/FStar
+          ref: master
+
+      - name: Try fetch built F*
+        id: cache-fstar
+        uses: actions/cache/restore@v4
+        with:
+          path: FStar
+          key: FStar-${{ runner.os }}-${{ runner.arch }}-${{ steps.checkout-fstar.outputs.commit }}
+
+      - name: Build F*
+        if: steps.cache-fstar.outputs.cache-hit != 'true'
+        run: |
+          make -C FStar ADMIT=1 -skj$(nproc) 1
+          make -C FStar ADMIT=1 -skj$(nproc) bootstrap
```

```suggestion
          make -C FStar ADMIT=1 -sj$(nproc) 1
          make -C FStar ADMIT=1 -sj$(nproc) bootstrap
```

There is no need continue building past the first error here.

---

Indeed thanks!